library/src/main/java/com/schibsted/spain/barista/permission/PermissionGranter.java

```diff
@@ -17,7 +17,8 @@
 public class PermissionGranter {
 
   private static final int PERMISSIONS_DIALOG_DELAY = 3000;
-  private static final int GRANT_BUTTON_INDEX = 1;
+  private static final String PERMISSIONS_DIALOG_ALLOW_ID = "com.android.packageinstaller:id/permission_allow_button";
+//    private static final String PERMISSIONS_DIALOG_DENY_ID = "com.android.packageinstaller:id/permission_deny_button";
```

Comment:
We will create another method that will use this ID to deny a permission. Thanks for it!